### DISCUSSION TITLE
how to use my own voice or my own set of speakers voice for text to speech

### DISCUSSION THREAD

**Discussion by souvikg544** [coqui-ai/TTS#1894] [General Q&A]
I have seen the github readme docs telling to use the following code :
$ tts --text "Text for TTS" --model_path path/to/config.json --config_path path/to/model.pth --out_path output/path/speech.wav
    --vocoder_path path/to/vocoder.pth --vocoder_config_path path/to/vocoder_config.json

and ---

$ tts --text "Text for TTS" --out_path output/path/speech.wav --model_path path/to/config.json --config_path path/to/model.pth --speakers_file_path path/to/speaker.json --speaker_idx <speaker_id>

But I am not able to understand how to build the models and use the config.json file. Any help with the steps involved will be very helpful.

**Comment by thorstenMueller** [comment]
Isn't that the same question you posted here? https://github.com/coqui-ai/TTS/discussions/1892
If so please try to focus on one thread as it makes it easier to answer.
